modules/onMath/index.js

```diff
@@ -46,7 +59,33 @@ module.exports.msg = function(text, from, reply, raw) {
 
   var res = MathScopeEval(text);
 
-  if(res !== null) {
+  // If res is null or just echos our input, filter it
+  if(res === null || res == text) {
```

Comment:
trim both (also toString() res first because it might not be a string) to catch some of the dumb spacing stuff.

Also, to catch text `hm/s`, res `hm / s` not being equal, maybe use `.replace(/\s/g, '')` instead of trim. I can't think of anything that breaks and it should fix that issue, but if you can think of something...